### ISSUE TITLE
A11y_Windows Terminal_Search or Find Operation_Screen Reader: After entering the text and performing search operation the screen reader is not reading whether that particular word is present or not in the terminal

### ISSUE THREAD

**Issue by SpandanaPaladi** [microsoft/terminal#5421]
#A11y_WindowsTerminal; #WindowsTerminal; #A11yMAS; #A11yTCS; #DesktopApp; #Win10; #C+E-H; #ScreenReader; #Narrator; #NVDA; #A11ySev2; #NonBenchMark; #MAS1.3.1

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-shcha1%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847087032&sdata=uWx6cTEfazn%2BoCV3cL1iYB4KN%2BuXQ/MTlmOVec%2BZgKc%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Version: 0.10.781.0

**Repro steps:**

1. Turn on screen reader
2. Hit the URL ""aka.ms/terminal"" and install ""Windows terminal (Preview)"" application
3. Open ""Windows terminal (Preview)"" application
4. Use "Alt + Shift + F" to perform search/find operation 
5. Verify whether the screen reader is reading that particular word which is searched is present in the terminal or not 

**Actual Result:**
Narrator/NVDA is not reading whether the searched word is present in the terminal or not

**Expected Result:​**
Narrator/NVDA should read whether the searched word is present in the terminal or not

**User Impact:​**
Screen Reader users will face difficulty if the screen reader don't read whether the searched word is present or not in the terminal because they don't understand whether the searched word is present or not

**MAS Reference:​**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug_Narrator not reading the searched word.pptx](https://github.com/microsoft/terminal/files/4502959/Bug_Narrator.not.reading.the.searched.word.pptx)
[Bug_NVDA not reading the searched word.pptx](https://github.com/microsoft/terminal/files/4502962/Bug_NVDA.not.reading.the.searched.word.pptx)

**Comment by carlos-zamora** [comment]
@SpandanaPaladi are you still seeing this issue, or can we close it?

**Comment by Neurrone** [comment]
when the search term is not found, pressing enter on the find dialog announces nothing. It would be ideal if some message was announced to the effect that it was not found.

When it is found, NVDA reads the search term. I find that this behaviour is not that intuitive. 

What does work well is that when escaping out of the find dialog, NVDA's review cursor is positioned at the starting position of the match for you to review around it.

**Comment by carlos-zamora** [comment]
@SpandanaPaladi are we compliant given the current state? Can we close this issue?

@Neurrone What you're requesting is a good idea, but I think with regards to tracking, it's slightly separate from what this issue is talking about. That said, I've opened issue #6319 to track it.

**Comment by Neurrone** [comment]
Looking at the issue title, I think it should remain open because nothing is read if no matches are found.

I agree that the UX for when text is found and how its conveyed should be in its own issue.

**Comment by Neurrone** [comment]
Oh, I see why you split it off in its own issue. I don't mind closing this or leaving this open while the proper UX is implemented in #6319.

**Comment by Amulya-95** [comment]
Now NVDA is reading regarding the searched keyword but Narrator is not reading anything. Still able to reproduce the issue with Narrator. And both NVDA and Narrator will not read anything no searched keyword is not available.

**Comment by Amulya-95** [comment]
Please provide the Sample app, so that we will file bug to Narrator team and will close the current bug as it is working fine with another free screen reader (NVDA)

**Comment by SpandanaPaladi** [comment]
Logged External bug https://microsoft.visualstudio.com/OS/_workitems/edit/27084958 to Narrator team.

Closing the current bug as it is working fine with NVDA
